src/components/Accounts/AddressesComponent.tsx

```diff
@@ -0,0 +1,160 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import { poppins } from 'pages/_app';
+import React, { useEffect, useState } from 'react';
+import BalanceDetails from './utils/BalanceDetails';
+import SendFundsComponent from './utils/SendFundsComponent';
+import AddressActionDropdown from './utils/AddressActionDropdown';
+import nextApiClientFetch from '~src/util/nextApiClientFetch';
+import { IAccountData } from '~src/types';
+import Address from '~src/ui-components/Address';
+import AddressComponent from './utils/AddressComponent';
+import { useUserDetailsSelector } from '~src/redux/selectors';
+import { Spin } from 'antd';
+
+const AddressesComponent = () => {
+	const userDetails = useUserDetailsSelector();
+	const { loginAddress } = userDetails;
+	const [accountData, setAccountData] = useState<IAccountData | null>(null);
+	const [isMultisigAddress, setIsMultisigAddress] = useState<boolean>(false);
+	const [isLoading, setIsLoading] = useState<boolean>(false);
+
+	const fetchData = async () => {
+		setIsLoading(true);
+		const { data, error } = await nextApiClientFetch<{ data: { account: IAccountData } }>('api/v1/accounts', { address: loginAddress });
+
+		if (error || !data) {
+			console.log('Error While fetching accounts');
+		}
+		if (data?.data?.account) {
+			setAccountData(data?.data?.account);
+			setIsMultisigAddress(data?.data?.account?.multisig?.multi_account_member?.length > 0);
+			setIsLoading(false);
+		}
+	};
+
+	useEffect(() => {
+		fetchData();
+	}, []);
+
+	return (
```

Comment:
add early return 
for ex if you accountData is not exist it should be show some kind of empty state